server/core/src/https/mod.rs

```diff
@@ -208,7 +208,8 @@ pub async fn create_https_server(
                 .route("/", get(|| async { Redirect::temporary("/ui") }))
                 .route("/manifest.webmanifest", get(manifest::manifest))
                 .nest("/ui", spa_router)
-                .layer(middleware::compression::new()) // TODO: this needs to be configured properly
+                .layer(middleware::compression::new())
+                .route("/ui/images/oauth2/:rs_name", get(oauth2::oauth2_image_get))
```

Comment:
Should this route be *above* compression? Or does it need to be after?

Reply:
Yeah I was tossing up between the load of attempting to compress the image again or not, since ... they should already be compressed - below the middleware removes the compression attempt which is probably better for now (testing to come later, but ... finger in the wind it's not beneficial.)